cli/cmd/install.go

```diff
@@ -442,7 +445,11 @@ var installCmd = &cobra.Command{
 			ControllerComponentLabel: k8s.ControllerComponentLabel,
 			CreatedByAnnotation:      k8s.CreatedByAnnotation,
 		}
-		return render(config, os.Stdout)
+		err := render(config, buf)
+		if err != nil {
+			return err
+		}
+		return InjectYAML(buf, os.Stdout)
```

Comment:
This is OK as long as we fix #303 in the same release that this lands in (i.e. either we need to fix #303 in for conduit 0.3, or we need to defer this for 0.4 and fix #303 in 0.4). In particular, we really don't want `conduit install` to output a bad or imcomplete configuration if InjectYAML fails partway through. However, we can resolve #303 in a separate PR.

Reply:
I can open a seperate PR for this right after I make these changes.